### DISCUSSION TITLE
HW 5 Last question

### DISCUSSION THREAD

**Discussion by caroline-cook** [cooperrc/computational-mechanics#66] [Q&A]
I was wondering if anyone knew roughly how long the animation takes to run? When I run it, the star remains instead of an input number, so I wasn't sure if it was for sure running or not.

**Comment by feliciones** [comment]
It should be running after a few seconds, are you getting any errors for your code?

**Comment by caroline-cook** [comment]
No I am not getting any errors. It just appears to keep "loading," as I get no output and the input number star doesn't go away.

**Reply by cooperrc** [reply]
If the `[*]` just keeps you waiting, you might have to click, "Kernel" -> "Restart & Run all"

Sometimes there's just a temporary bug/glitch and restarting seems to fix it.

**Reply by caroline-cook** [reply]
That worked, thank you!!
